org/blaste/index.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: org
+title: blaste
+site: http://www.blaste.net (in progress)
```

Comment:
You'll need to move the (in progress) comment elsewhere, as it confuses the system and tries to put it in the URL.